README.md

```diff
@@ -1,3 +1,5 @@
+![CI](https://github.com/nottoseethesun/Gridcoin-Tasks/workflows/CI/badge.svg)
```

Comment:
why is this linking to your personal repo?

Also the link goes to "no status"

Reply:
I think it's just my fork.

Reply:
It seems that perhaps in the over two years since I made this PR, GitHub has changed the Actions such that the old build is no longer here.